### PR TITLE
Add frontendbuild.sh script

### PR DESCRIPTION
**Pull request by ascott1**
The main purpose of this pull request is to add the `frontendbuild.sh` script, which will make a copy of the config file, run npm install, run bower install, and run a full set of grunt tasks. 

It also contains a little bit of cleanup of the package.json and stops inlining modernizr when developing locally.

## Review

@KimberlyMunoz

### CODE REVIEW

**Comment by ascott1** [comment]
Also, I'd like @imuchnik to take a look at this.

**Comment by ascott1** [comment]
It's worth noting that the package.json postinstall script duplicates some of the functionality of the frontendbuild.sh file. If this gets the thumbs up, should we suggest that users of the repo run frontendbuild.sh in the REAMDE in place of `npm install` and remove the postinstall script altogether?

**Comment by ascott1** [comment]
Last comment in my conversation with myself...

Is it possible to run this along with buildout? It would further simplify the initial application setup.

**Comment by cmc333333** [comment]
This is great, @ascott1 ! I think we should take a step back at some point and look at the three repos holistically to try to figure out a quick start path. The VM route is a good start; I wonder if it's not the double-down solution. Perhaps we should have the VM instructions in https://github.com/cfpb/eregulations and reference them in each of the repositories. The individual repos would just describe "custom" setup?

**Comment by ascott1** [comment]
> Perhaps we should have the VM instructions in https://github.com/cfpb/eregulations and reference them in each of the repositories. The individual repos would just describe "custom" setup?

That's what I've been envisioning as a sort of "ideal" state. I think it makes a lot of sense.

File: frontendbuild.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -ev
+
+if [ ! -f config.json ]; then
+  cp example-config.json config.json
+fi
```
Comment (imuchnik) [inline frontendbuild.sh]: I am not sure we need this.

File: Gruntfile.js [hunk 2]
```diff
@@ -181,7 +181,8 @@ module.exports = function(grunt) {
     */
   grunt.registerTask('nose', ['shell:nose-chrome', 'shell:nose-ie10']);
   grunt.registerTask('test', ['eslint', 'mocha_istanbul', 'nose']);
-  grunt.registerTask('build', ['squish', 'test']);
+  grunt.registerTask('test-js', ['eslint', 'mocha_istanbul']);
```
Comment (imuchnik) [inline Gruntfile.js]: 'test-js' task could call 'test' to make it [DRY](http://c2.com/cgi/wiki?DontRepeatYourself)-er

Comment (ascott1) [inline Gruntfile.js]: The nose tests that are in this repo are not set up by default and are very time consuming to run, so I think it's best not to include them initial the build. Any suggestions for circumventing those in frontendbuild.sh?

File: frontendbuild.sh (hunk 1 above)
Comment (ascott1) [inline frontendbuild.sh]: The grunt file consumes those settings and this should prevent it from erroring.

**Comment by imuchnik** [comment]
:+1:

**Pull request merged**